apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -260,39 +220,40 @@ export default class SummerWorkshop extends LabeledFormComponent {
       return (
         <div>
           <div id="regionalPartnerName">{this.renderRegionalPartnerName()}</div>
-          <p>
-            Teachers in this program are expected to attend the minimum number
-            of workshops that{' '}
-            <a
-              href="https://docs.google.com/document/d/1DhvzoNElJcfGYLrp5sVnnqp0ShvsePUpp3JK7ihjFGM/edit"
-              target="_blank"
-            >
-              correspond to the number of units they intend to teach
-            </a>
-            , including:
-          </p>
-          <ul>
-            <li>One summer workshop in {SUMMER_YEAR}</li>
-            <li>
-              Up to four one-day workshops during the {YEAR} school year
-              (typically held on Saturdays)
-            </li>
-          </ul>
+          {this.state.regionalPartnerName && (
+            <p>
+              Code.org’s Professional Learning Program is a yearlong program
+              starting in the summer and concluding in the spring. Workshops can
+              either be held in-person, virtually, or as a combination of both
+              throughout the year. Refer to the Regional Partner’s{' '}
+              <a
+                href={
+                  pegasus(
+                    '/educate/professional-learning/program-information'
+                  ) +
+                  (!!this.props.data.schoolZipCode
+                    ? '?zip=' + this.props.data.schoolZipCode
+                    : '')
+                }
+                target="_blank"
+              >
+                landing page
+              </a>{' '}
+              for more information about the schedule and delivery model.
+            </p>
+          )}
           {this.radioButtonsWithAdditionalTextFieldsFor('committed', {
             [TextFields.noExplain]: 'other'
           })}
           <div id="assignedWorkshops">
             {this.props.data.regionalPartnerId &&
               this.renderAssignedWorkshopList()}
           </div>
-          {this.radioButtonsFor('willingToTravel')}
-          We offer a series of virtual academic year workshops for those who are
-          unable to commit to traveling to in-person academic year workshops.
-          This workshop series consists of eight, 1.5 hour live sessions
-          throughout the academic year, hosted via a video conference tool.{' '}
-          <strong>Please note</strong> that this option is only available for
-          the academic year - all participants in the Professional Learning
-          Program must attend a summer workshop.
+          Code.org <em>may</em> offer a national series of virtual academic year
+          workshops to support teachers from regions who need to join a virtual
```

Comment:
This sentence is kind of awkward and very long. "teachers from regions who need to join a virtual academic year cohort" - can we remove the "from regions"? the regions aren't the ones joining the cohort.

Some possible alternatives for this sentence:

Code.org may offer a national series of virtual academic year workshops to support teachers who need to join a virtual academic year cohort in order to engage in the full Professional Learning Program because a virtual option is not offered in their region or is offered on a schedule that doesn’t work for them.

Code.org may offer a national series of virtual academic year workshops to support teachers who need to join a virtual academic year cohort because a virtual option is not offered in their region or is offered on a schedule that doesn’t work for them.

Reply:
oof I missed how awkward this phrasing was. @tess323 are you good if I clean this up a bit or do we need to ask anyone specific for approval?

Reply:
checking in with Tawny.  Suggested Clare's first option:

Code.org may offer a national series of virtual academic year workshops to support teachers who need to join a virtual academic year cohort in order to engage in the full Professional Learning Program because a virtual option is not offered in their region or is offered on a schedule that doesn’t work for them.

Reply:
using the first option until we hear otherwise.